### ISSUE TITLE
How to remove the thin line surrounding the edge of the object after background removal?

### ISSUE THREAD

**Issue by happyeungin** [xuebinqin/DIS#85]
I tried to remove the background of a product from a blue background. After that, the product has some blue color on the edge of the product. How can I remove it?

![2023-06-28_13-23-20](https://github.com/xuebinqin/DIS/assets/40191295/c5c33896-da18-4aa1-aa33-46e5b40a8499)

![mouse3](https://github.com/xuebinqin/DIS/assets/40191295/f88a7c4b-17d5-47db-ad52-ea79e3a08f2a)

**Comment by happyeungin** [comment]
Another example, there is also shadow remained after the removal

![2023-06-28_13-26-46](https://github.com/xuebinqin/DIS/assets/40191295/020bb8bf-eed8-4fb2-bc0e-424d10918f79)
![cart](https://github.com/xuebinqin/DIS/assets/40191295/b2755b49-e104-4e23-a9d7-e0f937be1baa)

**Comment by Mlegbder** [comment]
I'm having the same problem. Have you solved it yet?

**Comment by wilfrediscoming** [comment]
> I'm having the same problem. Have you solved it yet?

Yes I do, you will need to use another model - this won't work to remove shadow.
